modules/packages/ArgumentParser.chpl

```diff
@@ -449,165 +581,670 @@ module ArgumentParser {
       try!{addOption(name="dummyDashHandler", opts=["--"], numArgs=0);}
     }
 
-    proc _parsePositionals(arguments:[?argsD] string, endIdx:int) throws {
-      var endPos = argsD.low;
-      var idx = argsD.low;
-      debugTrace("Parsing Positionals...");
-      debugTrace(arguments:string + " " + endIdx:string);
-      for handler in _positionals {
-        var arg = _result.getReference(handler._name);
-        debugTrace("begin matching " + handler._name);
-        endPos = handler._match(arguments, idx, arg, endIdx);
-        idx = endPos;
-        debugTrace("got endPos " + endPos:string);
-        if idx == endIdx then break;
-      }
+    /*
+    Add an argument type to the parser. Argument types are also known as
```

Comment:
I think it's better if we leave out "type"

Add an argument to the parser. Arguments are also known as